vyper/ast.py

```diff
@@ -0,0 +1,278 @@
+from itertools import (
+    chain,
+)
+from typing import (
+    Any,
+    List as ListTyping,
```

Comment:
Maybe instead of doing it this way you might just use the module namespace for these

```python
only_empty_fields: typing.List[Any] = ...
```